test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/ModelBindingContextTest.cs

```diff
@@ -10,14 +10,14 @@ namespace Microsoft.AspNet.Mvc.ModelBinding.Test
     public class ModelBindingContextTest
     {
         [Fact]
-        public void GetChildModelBindingContext()
+        public void CreateChildBindingContext()
```

Comment:
While you're fixing the test name, please add something about expectations to it.